test/Market.test.ts

```diff
@@ -343,7 +343,6 @@ describe('Market', () => {
           hre,
           totalAmountOfSupply,
```

Comment:
the changes in this file are simply updating existing tests to rely on using the contract logic for calculations and thus abstracting away the impact of the price multiple from hard-coded values in tests -- something that we'd been meaning to do to all our tests anyway